code/modules/reagents/Chemistry-Machinery.dm

```diff
@@ -104,6 +104,14 @@ USE THIS CHEMISTRY DISPENSER FOR MAPS SO THEY START AT 100 ENERGY
 			stat |= NOPOWER
 	nanomanager.update_uis(src) // update all UIs attached to src
 
+/obj/machinery/chem_dispenser/proc/can_use(var/mob/living/silicon/robot/R)

```

Comment:
The code in the the initial can_use proc is almost exactly the same code that was previously in attackby() to prevent any silicons except MoMMIs and mediborgs from putting beakers into chem dispensers.

When I tested it locally, both MoMMIs and mediborgs were able to put beakers into chem dispensers, and service borgs could not put beakers into chem dispensers. Additionally, neither MoMMIs nor mediborgs could put beakers into brewers, soda dispensers, and booze dispensers, and service borgs could put beakers into brewers, soda dispensers, and booze dispensers.
It all seemed to work as it should.